docs(window): update initial ContainmentSelector documentation (#2001)
telerik/blazor-docs: components/window/position.md

## ContainmentSelector

> By default, users can drag and resize the Window without constraints. It is possible to restrict the Window's visual position to the boundaries of a specific container:

Reviewer: ```suggestion
By default, users can drag and resize the Window without constraints. You can restrict the Window's visual position to the boundaries of a specific container:
```
